IPython/html/static/tree/js/newnotebook.js

```diff
@@ -85,16 +85,20 @@ define([
                     url += "?kernel_name=" + kernel_name;
                 }
                 w.location = url;
-            },
-            function (error) {
-                w.close();
-                dialog.modal({
-                    title : 'Creating Notebook Failed',
-                    body : "The error was: " + error.message,
-                    buttons : {'OK' : {'class' : 'btn-primary'}}
-                });
-            }
-        );
+        }).catch(function (e) {
```

Comment:
Does this need to be `...then(func).catch(errfunc)` instead of `...then(func, errfunc)`, or do you just prefer that style? The error should be coming from the original promise, not the then() section, right?